samplesheet_generator.py

```diff
@@ -455,16 +454,11 @@ def main():
 	
 	if   (index_type.lower() == "dual"):
 		dual_indexes = True
-	elif (index_type.lower() == "simple"):
+	else (index_type.lower() == "simple"):
```

Comment:
Regarding arg parser, choices were added for `index_type` and not allowing any other option is exactly what they are supposed to do, so that should be ok.  `.lower()` can be also removed as only `simple` and `dual` options are allowed.

Fixed in https://github.com/tinavisnovska/samplesheet_generator/commit/00ce91479737598011d91f9e0d588e7ebc86ec78